src/dom.js

```diff
@@ -0,0 +1,18 @@
+const STRATEGIES = {
+  value: 'value',
+  html: 'html'
+}
+
+function fillTarget({ selector, value, ...params }) {
+  const strategy = params.strategy || STRATEGIES.value;
+  const element = document.querySelector(selector);
+
+  if (!element) return null;
+  if (strategy === STRATEGIES.value) element.value = value;
+  if (strategy === STRATEGIES.html) element.innerHTML = value;
```

Comment:
```suggestion
  if (strategy === STRATEGIES.html) element.innerText = value;
```

¿Crees queue puedan existir casos en los que se inserten datos que el innerHTML escape de alguna forma? Estoy pensando si se pueden inyectar valores que al hacerse por innerHTML se transformen en otra cosa